stdlib/hashtbl.mli

```diff
@@ -249,6 +249,20 @@ val of_seq : ('a * 'b) Seq.t -> ('a, 'b) t
     will appear in the table.
     @since 4.07 *)
 
+val of_list : ('a * 'b) list -> ('a, 'b) t
+(** [Hashtbl.of_list l] is a table built from the given bindings. The bindings
+    are added in the same order they appear in [l], using {!add}.
```

Comment:
`of_list` uses `add`, `of_seq` uses `replace`.  I don't know which one is better, but the inconsistency is unfortunate.

Reply:
Indeed. `add` seems more natural to me, but I agree we should use the same function for both.